plugins/services/src/js/components/modals/CreateServiceModal.js

```diff
@@ -456,7 +462,10 @@ class CreateServiceModal extends Component {
 
   handleServiceReview() {
     const errors = this.getFormErrors();
-    if (errors.length === 0) {
+    if (
```

Comment:
To make this more clear, maybe condense into a single expression reading "are there any not permissive errors?":

```
if (errors.filter(error => !error.isPermissive).length > 0) {}
```